### PR TITLE
implement SendInBlueApi for send mail

### PR DESCRIPTION
**Pull request by parth704131**
this pr solve issue812
complete implementation of sendinblue api
build the code and run it 
if any changes needed please inform me i will refactor.

### CODE REVIEW

File: src/main/java/alfio/manager/system/SendInBlueMailer.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        final var payload = new HashMap<String, Object>();
+        if (ArrayUtils.isNotEmpty(attachment)) {
+            addAttachments(payload, attachment);
+        }
+        payload.put("sender", Map.of("email", from, "name", fromName));
+        payload.put("to", Map.of("email",to));
+        payload.put("cc",createPersonalizations(cc));
+        payload.put("replyTo",config.get(ConfigurationKeys.MAIL_REPLY_TO).getRequiredValue());
+        payload.put("subject",subject);
+        payload.put("templateId",config.get(ConfigurationKeys.SENDINBLUE_TEMPLATE_ID).getRequiredValue());
```
Comment (syjer) [inline src/main/java/alfio/manager/system/SendInBlueMailer.java:60]: templateId? You are not sending the text and html

Comment (parth704131) [reply src/main/java/alfio/manager/system/SendInBlueMailer.java:60]: sendinblue provide feature like you make your own template in its site so there is not required html content and if you want it then i will change according that

Comment (syjer) [reply src/main/java/alfio/manager/system/SendInBlueMailer.java:60]: You should totally remove the enum `ConfigurationKeys.SENDINBLUE_TEMPLATE_ID` that you have added.

Then I think everything should be ok.

Comment (parth704131) [reply src/main/java/alfio/manager/system/SendInBlueMailer.java:60]: okay

**Comment by syjer** [comment]
hi @parth704131 , thank you for the PR and changes, I'll do a test and merge it.

**Comment by syjer** [comment]
hi @parth704131 , just noticed, you didn't modify the admin UI, how you were able to configure it and test it?

I can't accept this PR until the UI part is done too.

**Comment by parth704131** [comment]
hey @syjer i have to change only html file right or else i have to change js also and i don't know js so much so can you guide me which file i have to change because how is set data in enums that i don't know

**Comment by syjer** [comment]
@parth704131 , I think you mainly need to modify the html, maybe some js. I don't remember the files, but you can find them easily by searching `Mailjet.com` or `SMTP`

**Pull request closed without merging**